src/components/TextInput/BaseTextInput.js

```diff
@@ -253,6 +254,12 @@ function BaseTextInput(props) {
     ]);
     const isMultiline = props.multiline || props.autoGrowHeight;
 
+    const lineHeight = useMemo(() => {
+        return  Browser.isSafari() ? _.isArray(props.inputStyle) ?
+            _.find(props.inputStyle, function (f) { return f.lineHeight !== undefined; })?.lineHeight :
+            props.inputStyle?.lineHeight : Browser.isMobileChrome() ? height : undefined;
+    }, [props.inputStyle, height])
+       
```

Comment:
@ayazalavi Nested tertiary operators are very difficult to understand. I believe we had a rule to not use it. Can we make it if...else statement or something like that?

Reply:
@sobitneupane whats the hold up? My other proposals are waiting for this PR to be merged since new contributor here 😃. Please merge it today if possible. I have lost bonus already. Can you do something about that? Can PR reviewer recommend for bonus?

Reply:
@ayazalavi Let's try to get it merged today. 

> Can PR reviewer recommend for bonus?

I don't think we can do that.

Can you please add some comments on why this change is required and link the issue we are trying to solve in the comment?

Reply:
done, look at latest commit